tests/integration/cli/test_cli.py

```diff
@@ -138,6 +138,11 @@ def test_otx_ov_test(recipe: str, tmp_path: Path) -> None:
     Returns:
         None
     """
+    if recipe == "instance_segmentation/openvino_model.yaml":
```

Comment:
How about maintaining this issue more explicitly ? by using JIRA or Github issue